modelserving/k8s/llm-server.yaml

```diff
@@ -15,8 +15,21 @@ spec:
       serviceAccountName: llm-server
       containers:
       - name: llm-server
-        image: fake.registry/llm-image:latest
-
+        image: gcr.io/gcp-project-id/llamacpp-gemma3-12b-it:latest
```

Comment:
nit: Add a comment to update gcp-project-id

Reply:
Good call, I added a comment (I also updated this PR pretty substantially)